platform/marvell-armhf/sonic-platform-nokia/7215/scripts/nokia-7215init.sh

```diff
@@ -30,6 +30,9 @@ nokia_7215_profile()
 # Install kernel drivers required for i2c bus access
 load_kernel_drivers
 
+# Disable sysrq-trigger
+echo 0 > /proc/sys/kernel/sysrq
```

Comment:
@Pavan-Nokia As I understand, there is still a small window of opportunity during boot which may trigger sysrq commands?

Reply:
@Pavan-Nokia How did you rule out this window of time is NOT impacted on the MSFT prod scenario?

Reply:
@prgeor What we have learnt from our debug sessions is that the MSFT scripts start sending data only when they see a prompt. This change is in the platform init which runs during postinstall, before the getty service which enables the prompt. 

The only way to get full protection from sysrq is to make a sonic-linux-kernel change to disable this feature completely this would remove any window / chance for failure on any platform